src/moss_price/main.py

```diff
@@ -0,0 +1,94 @@
+import os
+import json
+from web3 import Web3
+import discord
+from discord.ext import commands, tasks
+
+BOT_TOKEN = os.environ["DISCORD_BOT_TOKEN"]
+MOSS_ADDRESS = '0xAa7DbD1598251f856C12f63557A4C4397c253Cea'
```

Comment:
Could we rename all references to `Moss` to `MCO2` instead?

In the event that Moss ever has more tokenized offset types, we should be referring to the specific token rather than the org that created the token - similar to how we refer to "BCT" instead of "Toucan"